Gemfile

```diff
@@ -2,6 +2,7 @@ source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 gem "hiera", :path => File.dirname(__FILE__), :require => false
 gem 'deep_merge', :require => false
+gem 'hocon', "~> 1.1.0"
```

Comment:
Well, the `~>` should mean it will pick up 1.1.1 (but not 1.2.0), so the bump may not be needed (unless it actually won't work with 1.1.0).